src/record/export/usecases/get.ts

```diff
@@ -44,8 +49,17 @@ export const getRecords = async (
           attachmentsDir,
         }),
     );
+    if (localRecords.length > 0) {
```

Comment:
NITS: 
You can increment `numOfProcessedRecords` without if block even if `localRecords.length` can be zero.

---
I was interested in the performance difference.
I checked the performance for 100,000 records. (5 times)
- with if block: 1m 17s
- without if block: 1m 16s
Certain but tiny differences 😄

Reply:
@mshrtsr 
I removed if block.

I also checked the performance with only the Javascript logic code:
Loop over 20.000.000 and execute:
1. `numOfProcessedRecords += localRecords.length`
2.
```
let nonExistRecord = true;
if (nonExistRecord && arr.length > 0) {
    nonExistRecord = false;
  }
```
In average of 5 times, it is 21ms and 18ms. That is around 14%. 
But it loop over 20.000.000 . It is a big number.
So I intended to keep this code as its current.
Please take a look at it.
Thank you!

Reply:
@tuanphamcybozu 
OK, our main focus is to keep our code simple, so let's keep this code as is.
Thank you.

---
FYI: I also checked with actual E2E behavior with 100,000 records

- 1. 1m 16s
- 2. 1m 15s

So, your testing in logic seems correct 👍 

As a performance technique, there is loop unrolling.
But it makes our code dirty.
https://en.wikipedia.org/wiki/Loop_unrolling